src/Uno.UI.Runtime.Skia.Tizen/TizenHost.cs

```diff
@@ -64,12 +64,6 @@ public TizenHost(Func<WinUI.Application> appBuilder, string[] args = null)
 			_appBuilder = appBuilder;
 			_args = args;
```

Comment:
I don't think the `_args` field is needed. It's only used in the constructor now. So it can be removed and we use `args` parameter instead.